mobile/src/libs/components/app/app.tsx

```diff
@@ -21,6 +22,13 @@ const App: React.FC = () => {
       SplashScreen.hide();
     }, SPLASH_SCREEN_HIDE_TIMEOUT);
 
+    const startPlayer = async (): Promise<void> => {
+      await TrackPlayer.setupPlayer();
+      await TrackPlayer.setRepeatMode(RepeatMode.Queue);
+    };
+
+    void startPlayer();
```

Comment:
let's follow the flow as with notifications - create a separate package, then create a meditation dir in slices, add initPlayer action here and the dispatch it in meditation screen